src/markdown-pages/docs/ecosystem/galasa-cli.md

```diff
@@ -0,0 +1,153 @@
+---
+path: "/docs/ecosystem/ecosystem-cli"
+title: "Using the Galasa CLI"
+---
+
+Submit and monitor Galasa test runs in you Galasa Ecosystem by using a command line interface (Galasa CLI). The Galasa CLI lets you submit and monitor test runs by using the same set of commands, regardless of the technology that you are using to run your pipeline. 
+
+[Download the CLI tool](https://github.com/galasa-dev/cli/releases) and [access the code](https://github.com/galasa-dev/cli) from the _cli_ repository in GitHub. 
+
+## Getting started 
+
+To use the CLI commands you must reference the Galasa bootstrap file or URL. You can do this either by setting the `--bootstrap` flag or the `GALASA_BOOTSTRAP`environment variable.
+
+There are two key commands - `runs prepare` and `runs submit`.
```

Comment:
There are 2 keys commands this release,  but the number of commands will massively increase,  so think this needs to be reworked.   Think there should be a page for the "runs" command and this page indexes them.

Reply:
Have created a top level page and then child pages for each runs command. I am just wondering how many commands there might end up being though -  I think that it might get very cluttered in the table of contents?

Reply:
This is now updated as per our discussion